### PR TITLE
add dependabot and GH action

### PR DESCRIPTION
**Pull request by jgomez720**

### CODE REVIEW

**Comment by jgomez720** [comment]
@jessfraz can you tell me what I did wrong (if anything)? I just need to test the `convert.py` file, so I took what was in the litterbox and deleted a lot that I was certain wasn't needed.

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        - name: Install dependencies
+            shell: bash
+            run: |
+            pip install -r requirements.txt
+
+        - name: Run scripts
+            shell: bash
+            run: |
+            FILE=$(find ${{ matrix.test-path }} -name "*.py")
+            python $FILE
```
Comment (jessfraz) [inline .github/workflows/build-test.yml:47]: you can change both these lines to just be
`python convert.py`

File: .github/workflows/build-test.yml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        strategy:
+        matrix:
+            python-version: [3.9]
+            test-path:
+            - ./
+
+        steps:
+        - uses: actions/checkout@v4
+            with:
+            lfs: true
```
Comment (jessfraz) [inline .github/workflows/build-test.yml:32]: Either the line above `with:` needs to be indented to the left under uses or `lfs:` needs to be indented more

**Comment by jessfraz** [comment]
nice!

**Pull request merged**